### ISSUE TITLE
Between tag still not working correctly

### ISSUE THREAD

**Issue by enxoco** [transformstudios/statamic-events#19]
I just created a fresh install of Statamic with version 3.3 of the addon.  I have copied over my events collection and created a blank page.  Using the tag from your docs:
```
{{ events:between collection="events" from="Jan 11 2022" to="Jan 12 2022" limit="5" paginate="true" }}
  {{ dates }}
    {{ start_date }} - {{ title }}<br>
  {{ /dates }}
  {{ pagination }}
    {{ if prev_page }}<a href="{{ prev_page }}">Previous</a>{{ /if }}
    {{ if next_page }}<a href="{{ next_page }}">Next</a>{{ /if }}
  {{ /pagination }}
{{ /events:between }}
```
The code above produces the following output:
![image](https://user-images.githubusercontent.com/34722899/149351652-96be028a-2ce7-42e6-b0df-d9b4c7426809.png)

This is the correct way that the tag should function, however it is still missing pagination information.  

This is what happens if I dump the tag. 

![image](https://user-images.githubusercontent.com/34722899/149351853-5a091039-886e-4195-b51a-7fb836f919bc.png)

I would expect to have the actual total number of results as well as the links to the previous/next pages.  If I manually go to page 3 then I see a previous link but still no total results or other links.

![image](https://user-images.githubusercontent.com/34722899/149351300-d9a68dc0-a997-463c-aa39-66b76ea9408d.png)

**Comment by edalzell** [comment]
The total results should be in the main group of data, not the paginate

**Comment by edalzell** [comment]
merged your PR and released in 3.3.2, thanks!
